_includes/footer.html

```diff
@@ -4,18 +4,20 @@
 		{% for member in site.data.members %}
 		<div class="group--member">
 			<a href="http://twitter.com/{{ member.twitter }}">
-				<img src="{{ member.gravatar }}" alt="{{ member.name }}" id="{{ member.name }}">
-			</a>
-			<h4>{{ member.name }}</h4>
-			<div class="social">
-				{% if member.url %}<a href="{{ member.url }}" class="genericon-link"></a>{% endif %}
-				{% if member.github %}<a href="http://github.com/{{ member.github }}" class="genericon-github"></a>{% endif %}
-				{% if member.codepen %}<a href="http://codepen.io/{{ member.codepen }}" class="genericon-codepen"></a>{% endif %}
-				{% if member.email %}<a href="mailto:{{ member.email }}" class="genericon-mail"></a>{% endif %}
-			</div>
-		</div>
-		{% endfor %}
-	</section>
+				<div class="avatar--wrapper">
+          <img src="{{ member.gravatar }}" alt="{{ member.name }}" id="{{ member.name }}">    
+        </div>
+      </a>
+      <h4>{{ member.name }}</h4>
+      <div class="social">
```

Comment:
Can't this be solved with a correct got config? Like the new line at the end thing.

> On 09 Mar 2016, at 09:44, Haroen Viaene <notifications@github.com> wrote:
> 
> In _includes/footer.html:
> 
> > -			<h4>{{ member.name }}</h4>
> > -			<div class="social">
> > -				{% if member.url %}<a href="{{ member.url }}" class="genericon-link"></a>{% endif %}
> > -				{% if member.github %}<a href="http://github.com/{{ member.github }}" class="genericon-github"></a>{% endif %}
> > -				{% if member.codepen %}<a href="http://codepen.io/{{ member.codepen }}" class="genericon-codepen"></a>{% endif %}
> > -				{% if member.email %}<a href="mailto:{{ member.email }}" class="genericon-mail"></a>{% endif %}
> > -			</div>
> > -		</div>
> > -		{% endfor %}
> > -	</section>
> > +				<div class="avatar--wrapper">
> > +          <img src="{{ member.gravatar }}" alt="{{ member.name }}" id="{{ member.name }}">    
> > +        </div>
> > +      </a>
> > +      <h4>{{ member.name }}</h4>
> > +      <div class="social">
> because all the rest probably already was spaces
> 
> —
> Reply to this email directly or view it on GitHub.
>